server/src/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -3761,6 +3767,8 @@ public boolean setupVmForPvlan(boolean add, Long hostId, NicProfile nic) {
         } catch (AgentUnavailableException e) {
             s_logger.warn("Agent Unavailable ", e);
             return false;
+        } catch (OperationCancelledException e) {
```

Comment:
it answer is null, false is returned at the end. removed return in the above exception handling.